src/cryptojwt/jwt.py

```diff
@@ -377,6 +377,10 @@ def unpack(self, token):
                 _msg_cls = self.iss2msg_cls[_info["iss"]]
             except KeyError:
                 _msg_cls = None
+                LOGGER.warning(
```

Comment:
If the KeyError is due to the fact that `_info['iss']` the exception handler will result in a new KeyError.

Reply:
Perhaps use the term "JWT" instead of "Json Web Token" :-)

Reply:
yes, better!